ccloud-observability/docs/general-scenarios/request-rate-limits.rst

```diff
@@ -11,7 +11,7 @@ client is throttled, |ccloud| will delay the client's requests for ``produce-thr
 producers or ``fetch-throttle-time-avg`` (in ms) for consumers
 
 |ccloud| offers different cluster types, each with its own `usage limits <https://docs.confluent.io/cloud/current/clusters/cluster-types.html#basic-clusters>`__. This demo assumes
-you are running on a "basic" or "standard" cluster; both have a request limit of 1500 per second.
+you are running on a "basic" or "standard" cluster; both have a request limit of 15000 per second.
```

Comment:
@cherylws to future proof this, can you remove the clause altogether so that we don't have to chase it again in the future.

Reply:
yes, that is a better solution